File: API/Models/Timesheets/TimesheetCard.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using API.Models.Users;

namespace API.Models.Timesheets
{
    public class TimesheetCard
    {
        [Key]
        public int TimesheetCardId { get; set; }
        public DateTime Date { get; set; }
        public float TotalTime { get; set; }
        public string Status { get; set; }
        public DateTime Created { get; set; } = DateTime.Now;

        public ICollection<TimesheetWeek> TimesheetWeeks { get; set; }

        public AppUser AppUser { get; set; }
        public int AppUserId { get; set; }

    }
}

File: client/src/app/_models/timesheetCard.ts
import { TimesheetWeek } from "./timesheetWeek";

export interface TimesheetCard {
    timesheetCardId: number;
    created: Date;
    date: Date;
    totalTime: number;
    status: string;
    timesheetWeeks: TimesheetWeek[];
    firstName: string;
    lastName: string;
}